Makefile

```diff
@@ -55,4 +55,4 @@ test: deps
 
 full-test: deps
 	@echo ⚙️ Running all tests...
-	@cargo +nightly t -- --include-ignored --test-threads=1
\ No newline at end of file
+	@cargo +nightly t -- --test-threads=1
```

Comment:
Why you've removed `--include-ignored`? `gclient`-based tests are with `#[ignore]` attribute.

Reply:
Exactly, my mistake.